src/Address/IPv6.php

```diff
@@ -50,6 +50,38 @@ class IPv6 implements AddressInterface
      */
     protected $rangeType;
 
+    /**
+     * The list of IPv6 RFC designated address ranges.
+     *
+     * @var mixed
+     */
+    public static $reservedRanges = array(
+        array('cidr' => '::/128',    'type' => RangeType::T_UNSPECIFIED),        //RFC 4291
+        array('cidr' => '::1/128',   'type' => RangeType::T_LOOPBACK),           //RFC 4291
+        array('cidr' => '100::/64',  'type' => RangeType::T_DISCARDONLY),        //RFC 4291
+        array('cidr' => '100::/8',   'type' => RangeType::T_DISCARD),            //RFC 4291
+        //['cidr' => '2002::/16', 'type' => RangeType::],                   //RFC 4291
```

Comment:
What are these commented lines?

Reply:
Commented out for clarity (maybe not clear), they are reserved ranges as defined in the RFC but as the default is T_RESERVED for IPv6 there's no need to perform the checks in the loop. So commenting them out is a performance improvement, if you like we could uncomment them for completeness.